Models/Interior/Panel/Instruments/clock/clock.xml

```diff
@@ -16,7 +16,21 @@
         <object-name>Face</object-name>
         <object-name>Case</object-name>
     </effect>
-
+    
+    <animation>
+        <type>select</type>
+        <object-name>Face</object-name>
+        <object-name>SecondHand</object-name>
+        <object-name>MinuteHand</object-name>
+        <object-name>HourHand</object-name>
+        <condition>
+            <equals>
+                <property>sim/model/c172p/aerotechlc2-visible</property>
```

Comment:
Can you rename this property to `digital-clock-visible`?

And doing an `<equals>` should not be necessary if it's a boolean. You can just wrap it in a `<not>`.

Reply:
I would rather make the whole model visible/hidden (in `c172p.xml`) instead of doing it here.

It's kinda ugly that the analog clock depends on the digital clock and vice versa. See https://en.wikipedia.org/wiki/Coupling_(computer_programming)